### DISCUSSION TITLE
Firefox can't disable screensaver

### DISCUSSION THREAD

**Discussion by ibahnasy** [netblue30/firejail#4404] [Q&A] [answered]
Firefox can't disable screensaver while playing Youtube video.
What firejail parameters I should use to prevent this?

**Comment by rusty-snake** [comment, marked as answer]
https://github.com/netblue30/firejail/blob/599b9af36aeb734f2577c49721e90f3d126fc1bf/etc/profile-a-l/firefox.profile#L53-L54
